interview-questions.md

```diff
@@ -8,59 +8,85 @@ Then, research the question to expand on your answer. Even if you feel you have
 1. What is git? What is the difference between git and Github?
 
   Your answer:
+Git is a repository track file system, meaning a git is created to track changes, updates, deletion of files. It allows for more than one coder to work on the same code by creating different branches, alterante paths, to work independently in the code and later on merge the changes to a single main path.
 
   Researched answer:
+Git is a version control system that allows many coders to work on the same code. 
+Github is a internet service provider that host version control using Git.
 
 
 
 2. What types of evaluations will return a Boolean value?
 
   Your answer:
+Conditional statements will return a boolean value, "true" and "false". The conditional statements use syntax such as if,
+elseif, else and  &&, ||, <, >. 
```

Comment:
More specifically, relational, logical, and equality operators will return Boolean values.